resume.page

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Meryl Streep- Top 10</title>
+  <link rel="stylesheet" href="stylesheet.css" type="text/css" media="screen">
+  <nav>
```

Comment:
All content that should be seen by the user in the browser window, like this nav, belongs in the body not the head. Please move. 🔨